cpp/unreal_plugins/SimulationController/Source/SimulationController/SonarSensor.cpp

```diff
@@ -1,56 +1,56 @@
-////------ BEGIN UE5 MIGRATION ------////
```

Comment:
Based on our most recent technical discussion, let's not check in `SonarSensor` for now.